### PR TITLE
Updating the hub_api for hub CLI

### PR DESCRIPTION
**Pull request by VamshiReddy02**
ref: [#3 ](https://github.com/itowlson/spin-hub-cli/pull/3)
In this PR, I added `template_id `and `repo_url` to the `hub_list_api`, which are very important for the Hub CLI.

### CODE REVIEW

File: content/api/hub/template_sqlite_explorer.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ last_updated = "2023-10-22T00:22:56Z"
 spin_version = ">=v1.4"
 summary =  "A Spin component for exploring the contents  and running a shell for SQLite stores."
 url = "https://github.com/karthik2804/spin-sqlite-web-cli"
+template_id = ""
```
Comment (itowlson) [inline content/api/hub/template_sqlite_explorer.md:17]:
```suggestion
template_id = "spin-sqlite-explorer"
```

File: content/api/hub/template_sqlite_explorer.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ last_updated = "2023-10-22T00:22:56Z"
...
+repo_url = ""
```
Comment (itowlson) [inline content/api/hub/template_sqlite_explorer.md:18]:
```suggestion
repo_url = "https://github.com/karthik2804/spin-sqlite-web-cli"
```

**Review by itowlson — CHANGES_REQUESTED** [review]
Looks good, just one place where some fields got left blank.

I also noticed that the Jekyll template (https://developer.fermyon.com/hub/preview/template_jekyll) didn't score an API entry.  Is this intentional?

**Comment by VamshiReddy02** [comment]
oopsie! I missed some templates, my bad.

**Review by itowlson — APPROVED** [review]
Looks good!  Thanks for this - I'm excited to see this being built out!

**Comment by itowlson** [comment]
I'm going to hold off merging so Karthik has the chance to take a look, since he built the original API machinery and is more familiar with it than I am.

**Review by karthik2804 — APPROVED** [review]

**Pull request merged**